src/pages/home/home--logged-in.stories.js

```diff
@@ -662,48 +637,37 @@ const Template = (args) => (
           title-level="2"
           title-text=""
```

Comment:
this link list has some properties which don't exist.
can you please globally check all <sbb-link-list> usages for its properties.